libraries/AP_HAL_ChibiOS/hwdef/CubeOrangePlus/hwdef.dat

```diff
@@ -50,6 +50,8 @@ SPIDEV icm42688_ext2  SPI4 DEVID4  GYRO_EXT_CS   MODE3  2*MHZ  8*MHZ
 #IMU 2
 SPIDEV icm45686       SPI1 DEVID4  ICM45686_CS   MODE1  2*MHZ  8*MHZ
 SPIDEV icm20649       SPI1 DEVID4  MPU_CS        MODE3  4*MHZ  8*MHZ
+SPIDEV icm45686_aux   SPI1 DEVID4  ICM45686_CS   MODE1  2*MHZ  8*MHZ
```

Comment:
it seems odd to have the same CS line, with same device type and devid listed twice. This would not be distinguishable from icm45686 in the parameter devids
maybe change the devid so we can distinguish them?

Reply:
@tridge it is actually the same IMU, different names for same are used because it can be tried two times, first it can be ignored as second instance, another is as aux IMU after three IMUs are detected.

Reply:
ok, but maybe if we used a different DEVIDn then we would better be able to tell from a log what is happening?